frontend/styles/globals.css

```diff
@@ -17,22 +17,25 @@ body {
   font-weight: 300;
 }
 
-{/* Enable correct behavior for sticky footer */}
+/* Enable correct behavior for sticky footer */
 #__next {
   display: flex;
   flex-direction: column;
   min-height: 100%;
 }
 
+a:hover {
+  text-decoration: none;
+}
+
 #footer a {
   color: white;
 }
 
-a:hover {
-  text-decoration: none;
+legend {
+  width: revert;
```

Comment:
This is what fixes the style problem. Since we import this file after importing bootstrap CSS, this resets to browser defaults. The `revert` is correctly overridden by other component-level styles.

Reply:
Should we add a comment here?